src/mysky/encrypted_files.ts

```diff
@@ -345,3 +416,27 @@ export function encodeEncryptedFileMetadata(metadata: EncryptedFileMetadata): Ui
 
   return bytes;
 }
+
+/**
+ * Validates the path seed and converts it to bytes.
+ *
+ * @param pathSeed - The given path seed.
+ * @returns - The path seed bytes.
+ */
+function validateAndGetFilePathSeedBytes(pathSeed: string): Uint8Array {
+  if (
```

Comment:
I know this is a bit of a `nit` comment but I think the validation error message should make a distinction between length and whether or not it's a hex string, and it's just good to be very verbose here so instead of the `!==` in combination with `&&` and `||` conditional, I think `includes` is just more readable.

```
if (!isHexString) {
    // throw custom validation error indicating it's not a valid hex string
}

const acceptedLengths = [
    ENCRYPTION_PATH_SEED_FILE_LENGTH, 
    ENCRYPTION_PATH_SEED_DIRECTORY_LENGTH
];
if (!acceptedLengths.includes(pathSeed.length) {
    throwValidationError(
      "pathSeed",
      pathSeed,
      "parameter",
      `a valid file or directory path seed of length '${ENCRYPTION_PATH_SEED_FILE_LENGTH}' or '${ENCRYPTION_PATH_SEED_DIRECTORY_LENGTH}'`
    );
}
```